Readme.md

```diff
@@ -100,15 +99,58 @@ mod1    0.0.1
 module2 0.2.0  
 ```
 
-### Wrapping Column Cells
+### Filtering & Ordering Columns
+
+By default, all properties are converted into columns, whether or not
+they exist on every object or not.
+
+To explicitly specify which columns to include, and in which order,
+supply a "columns" or "include" array ("include" is just an alias).
+
+```javascript
+var data = [{
+  name: 'module1',
+  description: 'some description',
+  version: '0.0.1',
+}, {
+  name: 'module2',
+  description: 'another description',
+  version: '0.2.0',
+}]
+
+var columns = columnify(data, {
+  columns: ['name', 'version'] // note description not included
```

Comment:
I wonder if the comment is needed, considering how the text around the code and the example explains it well.